modules/lm-coursier/src/main/scala/lmcoursier/CoursierConfiguration.scala

```diff
@@ -41,6 +44,11 @@ import xsbti.Logger
   reconciliation: Vector[(ModuleMatchers, Reconciliation)] = Vector.empty,
   @since
   classpathOrder: Boolean = true,
+  @since
+  verbosityLevel: Int = 0,
+  ttl: Option[Duration] = CacheDefaults.ttl,
+  checksums: Vector[Option[String]] = CacheDefaults.checksums.to[Vector],
+  cachePolicies: Vector[String] = CacheDefaults.cachePolicies.to[Vector].map(CoursierConfiguration.cachePolicyToString),
```

Comment:
Let's add an ADT for that under `lmcoursier.definitions`, like [this one](https://github.com/coursier/sbt-coursier/blob/aa83893027725900e97fe01411ea7a976739e75e/modules/lm-coursier/src/main/scala/lmcoursier/definitions/Reconciliation.scala), rather than using `String`s.

Reply:
"that" being.. CachePolicy ? OK

Reply:
Yes, a kind of duplicate of `CachePolicy` under `lmcoursier.definitions`. The original `CachePolicy` gets shaded, but the one under `lmcoursier.definitions` wouldn't and would still be usable for sbt users.

Reply:
Done. There is now a FromCoursier object so that we can pull the CoursierDefault and expose them to users across the bulkhead